sonarqube/tests/test_integration.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from .metrics import WEB_METRICS
+
+pytestmark = [pytest.mark.integration, pytest.mark.usefixtures('dd_environment')]
+
+
+def test_integration_service_check_ok(aggregator, dd_run_check, sonarqube_check, web_instance_with_components):
+    # Given
+    check = sonarqube_check(web_instance_with_components)
+    # When
+    dd_run_check(check)
+    # Then
```

Comment:
We should remove these comments before merging

Reply:
They are expressly written comments to distinguish the different parts of the test.
[GivenWhenThen](https://martinfowler.com/bliki/GivenWhenThen.html)

Reply:
I understand that, but my assertion is that this is not helpful and extremely uncommon, not just in Python but generally

Reply:
> I do not expect this as a 'rule-to-follow' for everyone

ok then 👍